easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.032-intel-2015b-Perl-5.20.3.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PerlModule'
+
+name = 'DBD-mysql'
+version = '4.032'
+
+homepage = 'http://search.cpan.org/~capttofu/DBD-mysql-%(version)s/'
+description = """Perl binding for MySQL"""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
```

Comment:
I'm not going to start installing Perl modules with subtoolchains (at least not until @ocaisa's subtoolchain stuff is working)